### PR TITLE
GEOMESA-1621 Adding JAR lookup in KAFKA_HOME, adding default zkPath, …

### PR DESCRIPTION
**Pull request by elahrvivaz**
…adding logger install

Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

File: geomesa-kafka/geomesa-kafka-tools/bin/geomesa-kafka [hunk 1]
```diff
@@ -16,9 +16,14 @@ fi
 # Load common functions and setup
 . "${%%gmtools.dist.name%%_HOME}"/bin/common-functions.sh
 
+if [ -n "$KAFKA_HOME" ]; then
```
Comment (aheyne) [inline geomesa-kafka/geomesa-kafka-tools/bin/geomesa-kafka]: [[Picky]] Double brackets

File: geomesa-kafka/geomesa-kafka-tools/bin/install-loggers.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  # get stuff
+  for x in "${urls[@]}"; do
+    fname=$(basename "$x");
+    echo "fetching ${x}";
+    wget -O "${1}/${fname}" "$x" || { rm -f "${1}/${fname}"; echo "Failed to download: ${x}"; \
+      errorList="${errorList} ${x} ${NL}"; };
+  done
+
+  if [[ -n "${errorList}" ]]; then
+    echo "Failed to download: ${NL} ${errorList}";
```
Comment (aheyne) [inline geomesa-kafka/geomesa-kafka-tools/bin/install-loggers.sh]: I just realized this should be ${errorList[@]} everywhere (not just this file). Should this be fixed in another pr?

Comment (elahrvivaz) [inline geomesa-kafka/geomesa-kafka-tools/bin/install-loggers.sh]: yeah, could you open a new pr with the fixes?

Comment (aheyne) [inline geomesa-kafka/geomesa-kafka-tools/bin/install-loggers.sh]: fixed in https://github.com/locationtech/geomesa/pull/1316
